hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -1548,6 +1548,13 @@
     <value>600000</value>
   </property>
 
+  <property>
+    <description>Whether to clean up nodemanager logs when log aggregation is enabled
```

Comment:
We must warn the users. Could you document that setting to false disables the cleanup nodemanager logging, and it causes disk full in the long run. Users can set to false for test-only purpose.

Reply:
Updated the description in my next commit.